### ISSUE TITLE
Wiki addition

### ISSUE THREAD

**Issue by SmallPepperZ** [arxenix/better-shulkers#9]
I made a fork of this repo, and added a wiki for the different enchantments, splicing in cuts of Xisuma's video where he talks about each enchantment. I wasn't sure how to do a pull request for a wiki, so here is a link to my fork, https://github.com/smallpepperz1/better-shulkers. Let me know if I need to do something so you can merge them.

**Comment by rmobis** [comment]
Did you delete the fork? I'm contemplating the idea of developing a similar mod and having a good wiki (or at least something to base it on) would be amazing.

**Comment by SmallPepperZ** [comment]
> Did you delete the fork? I'm contemplating the idea of developing a similar mod and having a good wiki (or at least something to base it on) would be amazing.

yeah, I deleted it because this mod's author seems to have stopped working on it, I still have a different layout for it, which I'll add you to

**Comment by SmallPepperZ** [comment]
https://github.com/SmallPepperZPublic/better-shulkers-wiki

**Comment by rmobis** [comment]
Yeah, I noticed it too. I'm thinking of writing a mod to implement @gnembon's take on this. He has a [video](https://youtu.be/sEQPMNN5UGk) explaining it and expanding on some of the concepts from the original video and he also implemented it quite flawlessly through a [scarpet script](https://github.com/gnembon/scarpet/blob/master/programs/survival/shulkerboxes.sc).

However, because it is not a mod and just a script, it doesn't add the actual enchantments (also doesn't have enlarge) and works based on the shulker's name. So I think I'll the mod to fill in the gap and have been taking a look at the suggestions from here. Stay tuned. =D

Oh, and thanks for the link, I'll check that out.
